packages/remix-dev/compiler.ts

```diff
@@ -353,7 +354,8 @@ async function createBrowserBuild(
     browserRouteModulesPlugin(config, /\?browser$/),
     emptyModulesPlugin(config, /\.server(\.[jt]sx?)?$/),
     NodeModulesPolyfillPlugin(),
-  ];
+    yarnPnpPlugin(),
+  ].filter((p): p is esbuild.Plugin => Boolean(p));
```

Comment:
No need to filter this array anymore now that we're always including `@yarnpkg/esbuild-plugin-pnp`

```suggestion
  ];
```